cloudos/queue/queue.py

```diff
@@ -128,12 +128,12 @@ def fetch_job_queue_id(self, workflow_type, batch=True, job_queue=None):
             default_queue_name = available_queues[-1]['label']
             queue_as_default = 'most recent suitable'
         if job_queue is None:
-            print(f'\tNo job_queue was specified, using the {queue_as_default} queue: ' +
+            print(f'[Message] No job_queue was specified, using the {queue_as_default} queue: ' +
```

Comment:
Non-blocking: We should look into having a better logger implementation that can prefix these values so in future you can call `logger.message`, `logger.warning` instead of manually adding the prefix